orttraining/orttraining/test/training_ops/cpu/tensor/gather_grad_op_test.cc

```diff
@@ -105,7 +105,7 @@ TEST(GatherGradOpTest, Gather_axis1_float_impl2) {
   int64_t output_shape = 4;
   RandomValueGenerator random{};
   std::vector<float> grad(random.Uniform<float>({axis_0 * axis_1 * axis_2}, 1.0f, 1.0f));
-  std::vector<int64_t> indices(random.Uniform<int64_t>({axis_1 * axis_2}, (int64_t)0, (int64_t)3));
+  std::vector<int64_t> indices(random.Uniform<int64_t>({axis_1 * axis_2}, (int64_t)0, (int64_t)2));
```

Comment:
nit: prefer to avoid C-style casts (https://google.github.io/styleguide/cppguide.html#Casting)

Reply:
Good idea! Love static cast syntax in addition to other benefits :)